lib/membrane_h264_plugin/parser/au_timestamp_generator.ex

```diff
@@ -0,0 +1,116 @@
+defmodule Membrane.H264.Parser.AUTimestampGenerator do
+  @moduledoc false
+
+  alias Membrane.H264.Parser.NALu
+
+  @vcl_nalus NALu.vcl_nalus()
+
+  @type t :: %{
+          au_counter: non_neg_integer(),
+          key_frame_au_idx: non_neg_integer(),
+          previous_vcl_nalu: NALu.t() | nil,
+          prev_pic_order_cnt_msb: integer()
```

Comment:
Can it be negative integer?

Reply:
I think so, but I'd have to analyze the spec to be sure :P